src/main/java/com/enonic/xp/app/users/rest/AdminCmsRestActivator.java

```diff
@@ -0,0 +1,34 @@
+package com.enonic.xp.app.users.rest;
+
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+
+import com.enonic.xp.jaxrs.JaxRsService;
+import com.enonic.xp.jaxrs.JaxRsServiceFactory;
+import com.enonic.xp.web.dispatch.DispatchConstants;
+
+@Component(immediate = true)
+public final class AdminCmsRestActivator
+{
+    private JaxRsService service;
+
+    @Activate
+    public void activate()
+    {
+        this.service.init();
+    }
+
+    @Deactivate
+    public void deactivate()
+    {
+        this.service.destroy();
+    }
+
+    @Reference
+    public void setService( final JaxRsServiceFactory factory )
+    {
+        this.service = factory.newService( "v2", "/admin/v2/rest", DispatchConstants.XP_CONNECTOR );
```

Comment:
I think, it is a collision here. If all apps try to register JaxRsService "v2" service, on the same path, only one service would work.
So, paths service paths, and groups should be different

Reply:
Strange thing, it didn't work at all when I changed groups and paths to be different in all apps. 
What I did was:
In AdminCmsRestActivator.java
`this.service = factory.newService( "v2users", ResourceConstants.REST_ROOT, DispatchConstants.XP_CONNECTOR );`
where
`String REST_ROOT = "admin/v2/rest/users/";`
And then in resource:
`@Component(immediate = true, property = "group=v2users")`